example-app/src/main/AndroidManifest.xml

```diff
@@ -4,6 +4,8 @@
     package="com.adyen.checkout.example">
 
     <uses-permission android:name="android.permission.INTERNET" />
+    <!--  add this permission when tried on api level 25 emulator for redirect app crashed for not granting this permission so added it  -->
+    <uses-permission android:name="android.permission.WAKE_LOCK" />
```

Comment:
I'm a bit skeptical about this 🤔 Did you find any resource that says `WAKE_LOCK` is required for redirects?

Reply:
Does this happen with the old method as well?

Reply:
yes it happens when you just click on `start` button in dropIn